src/store/substrate/index.js

```diff
@@ -246,6 +246,7 @@ export default {
         commit("SET_LAB_ACCOUNT", null)
         commit("SET_IS_LAB_ACCOUNT_EXIST", false)
         const labAccount = await queryEntireLabDataById(state.api, state.wallet.address)
+        console.log("labAccount", labAccount)
```

Comment:
please remove this part if not necessary

Reply:
updated @hilyds